stix_shifter_modules/elastic_ecs/stix_translation/json/beats_from_stix_map.json

```diff
@@ -184,7 +184,7 @@
       "severity": ["event.severity"],
       "start": ["event.start"],
       "timezone": ["event.timezone"],
-      "type": ["event.type.keyword"],
```

Comment:
Do we have `event_type` in `x-oca-event`, or `category`?

Reply:
Shall we also check whether we can use the `category` attribute?

Reply:
By definition, it’s not the same thing. But considering `category` is an array of string, it can be used. How frequently is category used in practice?

Reply:
I think we don't have statistics about the popular data regarding `event.type`. We may raise this as a question tomorrow and get others' thoughts and understandings on `category`.

Reply:
The [kind](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html), [type](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html) and [category](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html) fields of the event category hierarchy have been consolidated in `x-oca-event:category`. See [0bae039](https://github.com/opencybersecurityalliance/stix-shifter/pull/1501/commits/0bae039195ea945d9ca27477c4601db150f19300).